packages/blockly-field-date/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@blockly/field-date",
+    "version": "3.20200304.0",
+    "description": "A Blockly date picker field that uses the Google Closure date picker.",
+    "scripts": {
+        "build": "gulp build"
+    },
+    "main": "build/date_compressed.js",
```

Comment:
Might be nice to stay consistent with build vs dist across packages. Not sure it matters a lot for the end user but since we are just starting out might be nice.

Reply:
Curious why we chose dist?

Reply:
From what I saw dist is for files that are going to be distributed, build is not production ready. If you have strong opinions for using build I can also update workspace search to make it consistent.